cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
+# All providers that support C&U support Metering Reports.SCVMM doesn't support C&U.
```

Comment:
@izapolsk , That's because SCVMM doesn't support C&U, so C&U metrics are not collected for SCVMM.

Reply:
I understand. But there are many tests in this file and only one or two have uncollectif(provider="scvmm") statements.

Reply:
@izapolsk , There are no uncollectif(provider="scvmm") statements in this module.

I have this statement in the module because of which, tests for only providers with the required field in cfme_data.yaml , cap_and_util are generated.

pytest.mark.provider([CloudInfraProvider], scope='module',
                         required_fields=[(['cap_and_util', 'test_chargeback'], True)]),